README.md

```diff
@@ -1,3 +1,5 @@
+>ℹ️&nbsp;&nbsp;SignalFx was acquired by Splunk in October 2019. See [Splunk SignalFx](https://www.splunk.com/en_us/investor-relations/acquisitions/signalfx.html) for more information.
```

Comment:
Do we want to have these notices in the first or the last line of the README? splunk-otel-java has this in the last line